src/Field/NominatimSearch/NominatimSearch.spec.jsx

```diff
@@ -143,11 +159,12 @@ describe('<NominatimSearch />', () => {
       //SETUP END
 
       const wrapper = TestUtil.mountComponent(NominatimSearch, {map});
-      const fitSpy = sinon.spy(map.getView(), 'fit');
+      const fitSpy = jest.spyOn(map.getView(), 'fit');
       wrapper.props().onSelect(item, map);
-      expect(fitSpy.calledOnce).toBeTruthy();
-      expect(fitSpy.calledWith(tranformedExtent)).toBeTruthy();
-      fitSpy.restore();
+      expect(fitSpy).toHaveBeenCalled();
+      expect(fitSpy.mock.calls[0][0]).toEqual(transformedExtent);
```

Comment:
We might better use `.toHaveBeenCalledWith`  https://facebook.github.io/jest/docs/en/expect.html#tohavebeencalledwith

(We should at least check if it works with the array and `toEqual`)